uai2017/process/commentProcess.js

```diff
@@ -0,0 +1,161 @@
+function(){
+    console.log('comment process initiated')
+    var or3client = lib.or3client;
+    
+    var list = note.invitation.replace(/_/g,' ').split('/');
+    list.splice(list.indexOf('-',1));
+    var conference = list.join(' ')
+
+    var getAuthorEmails = function(origNote){
+      console.log('get author emails initiated')
+      var origNoteAuthors = origNote.content.author_emails.trim().split(",");
```

Comment:
replace "author_emails.trim()" with "authorids" (without trimming) across all process functions.

you can see the most up-to-date version of the ICLR 2017 process functions to see this